TypeCobol.Test/Parser/Programs/TypeCobol/LibraryFunctionDeclarativeCheck.rdzPGM.txt

```diff
@@ -1,5 +1,5 @@
 ﻿--- Diagnostics ---
-Line 38[9,12] <27, Error, Syntax> - Syntax error : Inside a library only function declaration or declaratives are allowed  / sentence-1
+Line 38[9,12] <27, Error, Syntax> - Syntax error : A program which contains public procedure must have INIT-LIBRARY as first paragraph.
 
```

Comment:
It would be better to have a test for each case :
- first paragraph is not INIT-LIBRARY
- invalid declarations in public library
- missing INIT-LIBRARY

In fact the second case corresponds to SECTION declarations
```cobol
MySection SECTION.
    DISPLAY "1"
```

Reply:
fixed; new commit:
https://github.com/TypeCobolTeam/TypeCobol/tree/4e9f45bcbfdc878acbe170739c3bbd60dcc8a94a